core/tests/protractor_utils/ExplorationEditorImprovementsTab.js

```diff
@@ -154,6 +155,14 @@ var ExplorationEditorImprovementsTab = function() {
     return task.all(actionButtonLocator);
   };
 
+  this.verifyOutstandingTaskCount = (expectedTaskCount) => {
+    expect(taskCount.getText()).toMatch(String(expectedTaskCount));
+  };
+
+  this.verifyNoOutstandingTasks = () => {
+    expect(taskCount.getText()).toMatch('');
```

Comment:
As mentioned, please use the length approach:

```javascript
expect(taskCount.length).toEqual(0);
```

Reply:
As it turns out, the length approach cannot be used as `taskCount.length` is `undefined` and not equal to `0`. 
Instead, I'm using 
```js
expect(taskCount.isDisplayed()).toBe(false)
```